src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/SpringFoxConfig.kt

```diff
@@ -89,4 +102,47 @@ class SpringFoxConfig {
       }
     }
   }
+
+  // Fixes incompatibility with latest spring-boot: https://github.com/springfox/springfox/issues/3462
```

Comment:
Maybe add "PUD-1231" to this comment ... as a ticket to do this swagger migration/uplift.